src/cryptsetup/cryptsetup-tokens/cryptsetup-token-systemd-tpm2.c

```diff
@@ -234,6 +240,7 @@ _public_ void cryptsetup_token_dump(
         crypt_log(cd, "\ttpm2-policy-hash:" CRYPT_DUMP_LINE_SEP "%s\n", policy_hash_str);
         crypt_log(cd, "\ttpm2-pin:         %s\n", true_false(flags & TPM2_FLAGS_USE_PIN));
         crypt_log(cd, "\ttpm2-salt:        %s\n", true_false(salt));
+        crypt_log(cd, "\ttpm2-srk:         %s\n", true_false(primary_buf));
```

Comment:
hmm, so towards the user the concept is called "srk", and internally we call it "primary"? I think we should stick to one name here.